give.spec

```diff
@@ -1,24 +1,22 @@
-%define realname give
-%define realversion 3.1-5
-
 #don't strip binaries
 %define __os_install_post /usr/lib/rpm/brp-compress
 %define debug_package %{nil}
 
-Name: %{realname}-%{realversion}
-Version: ptools
-Release: 1
+#Name: give-%{version}-%{release}
+Name: give
+Version: 3.1
+Release: 5%{?dist}
 Summary: lc file transfer utility
 License: LLNL Internal
 Group: System Environment/Base
-Source0: %{realname}-%{realversion}.tgz
-BuildRoot: %{_tmppath}/%{name}-buildroot
+Source0: https://github.com/hpc/give/archive/v3.1-5.tar.gz
+BuildRoot: ~/rpmbuild/BUILDROOT
```

Comment:
The buildroot doesn't need to be specified. We can just remove this declaration.